src/operator/tensor/elemwise_unary_op.cc

```diff
@@ -388,14 +368,13 @@ Example::
    sqrt([4, 9, 16]) = [2, 3, 4]
 
 )code" ADD_FILELINE)
-.set_attr<FCompute>("FCompute<cpu>", UnaryCompute<cpu, mshadow_op::square_root>)
 .set_attr<nnvm::FGradient>("FGradient", ElemwiseGradUseOut{"_backward_sqrt"});
 
-MXNET_OPERATOR_REGISTER_BINARY(_backward_sqrt)
-.set_attr<FCompute>("FCompute<cpu>", BinaryCompute<cpu, unary_bwd<mshadow_op::square_root_grad> >);
+MXNET_OPERATOR_REGISTER_BINARY_LAUNCH_CPU_DR(_backward_sqrt,
```

Comment:
There's a more efficient way to have a dense result than to cast back and compute. This is implemented with binary and scalar computes, but not yet with unary.  Can remove for now.